documentation/tutorials/password-change.md

```diff
@@ -0,0 +1,260 @@
+        as: "user",
+        # Only necessary when using field policies with `private_fields :include`.
+        prepare_source: fn changeset ->
+          %{
+            changeset
+            | data:
+                Ash.load!(changeset.data, :hashed_password,
+                  context: %{private: %{password_change?: true}}
+                )
+          }
+        end,
+        actor: user
+      )
+
+    assign(socket, form: to_form(form))
+  end
+```
+`update/2` is covered in the `Phoenix.LiveComponent` life-cycle documentation, so we won't go into it here.  The private function `assign_form/1` should look familiar if you have any forms for Ash resources in your application, but with a significant addition: the `prepare_source` option.
+
+When using field policies, the `@current_user` assign set by AshAuthentication will not contain the value of the `hashed_password` attribute, because it is a private field.  This is what you normally want in your application, but the `:change_password` action needs this to validate the `:current_password` argument, so we need to explicitly load this attribute.  We also pass a private context field to the load operation to use in a field policy that we will write later on.
+
+The attribute `phx-target={@myself}` on the form in our template ensures the submit event is received by the component, so the `handle_event/3` function in this module is called, rather than the LiveView that mounts this component receiving the event.
+
+```elixir
+  @impl true
+  def handle_event("save", %{"user" => user_params}, %{assigns: assigns} = socket) do
+    case AshPhoenix.Form.submit(assigns.form, params: user_params) do
+      {:ok, user} ->
+        assigns.on_saved.()
+        {:noreply, socket}
+
+      {:error, form} ->
+        {:noreply, assign(socket, form: form)}
+    end
+  end
+```
+
+Again, this should look familiar if you have any forms on your other application resources, but handle the success case a little differently here, calling the function passed by the parent LiveView via the `on_saved` attribute.  This will make more sense when we use this component in a LiveView.
+
+Since the password-change workflow is done entirely in our application code, the [AshAuthentication policy bypass](https://hexdocs.pm/ash_authentication/policies-on-authentication-resources.html) will not pass.  We need to add a policy that allows a user to run the `:change_password` action on themselves.
+```elixir
+# lib/my_app/accounts/user.ex
+  # ...
+  policies do
+    # ...
+    policy action(:change_password) do
+      description "Users can change their own password"
+      authorize_if expr(id == ^actor(:id))
+    end
+  end
+  # ...
+```
+
+If you are using field policies, we also need to allow `ChangePasswordComponent` to read the `hashed_password` field, as we mentioned earlier.  We need to write a custom check to read the private context we set in the component earlier.
```

Comment:
```suggestion
If you are using field policies (with `private_fields :hide` or `private_fields :include`), we also need to allow `ChangePasswordComponent` to read the `hashed_password` field, as we mentioned earlier.  We need to write a custom check to read the private context we set in the component earlier.
```

Reply:
This doesn't work at all with `private_fields :hide`, unless you use `authorize: false` on the load operation.

Reply:
I have now moved everything covering field policies to a section at the end.